pkg/gardenlet/operation/botanist/cleanup.go

```diff
@@ -138,7 +139,7 @@ func cleanResourceFn(cleanOps utilclient.CleanOps, c client.Client, list client.
 				if utilclient.AreObjectsRemaining(err) {
 					return retry.MinorError(helper.NewErrorWithCodes(err, gardencorev1beta1.ErrorCleanupClusterResources))
 				}
-				return retry.SevereError(err)
+				return retry.SevereError(fmt.Errorf("failed cleanup of resource kind %s : %w", list.GetObjectKind(), err))
```

Comment:
I think we have to use `obj.GetObjectKind().GroupVersionKind().Kind` instead of `list.GetObjectKind()`. `list.GetObjectKind()` returns `schema.ObjectKind` which is interface. I am not sure if it has a proper String implementation. 

See example usage for `obj.GetObjectKind().GroupVersionKind().Kind`: https://github.com/gardener/gardener/blob/485c25124ea536e4610c627109017dd34d434921/pkg/controller/reference/reconciler.go#L179-L184

Reply:
`list.GetObjectKind().GroupVersionKind().Kind` would be the equivalent one, right?